src/models/hashmodel.h

```diff
@@ -104,6 +104,23 @@ class HashModel : public QAbstractTableModel
         return index(row, column);
     }
 
+    static QString formatSymbolBinaryString(const QString& symbol, const QString& binary)
```

Comment:
once you use tr() templates, these function will have to change accordingly. I guess we want:

- a helper that returns the input string or "??" when that is empty
- a helper that formats the cost into a fractional string

such that it gets used like this:

```
tr("%1 in %2\nself cost: %3 (%4 of total of %5)\ninclusive cost: ...").arg(formatString(symbol), formatString(binary), formatCost(selfCost), formatCostRelative(selfCost, totalCost), formatCost(totalCost), ...);
```

the advantage here is that this is properly translatable, esp. for RTL languages